modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -111,20 +109,25 @@ $ oc apply -f httpbin-vs.yaml
 $ oc create namespace curl
 ----
 
-. Deploy the `curl` client by applying the curl.yaml file:
+. Deploy the `curl` client by applying the `curl.yaml` file:
```

Comment:
Same situation as earlier with this command. IMO, now that I'm reading a second command just like this, I feel like you can drop the "by applying the X yaml file" phrase from both of these steps. 

It seems like the name of the YAML file is not relevant given that it's only part of the filepaths in these commands, and users otherwise might not need to know about the file names more than the action they're taking

Reply:
done